cis_env.py

```diff
@@ -3,6 +3,7 @@
 import numpy as np
 import dna_decoding
 from cis_cell import random_vector_of_length
+import scipy.constants as cons
```

Comment:
If you want to use scipy in CIS you have to add it to the dependencies. But maybe you also find a more elegant way to import the speed of light :wink:.